Documentation/Examples.md

```diff
@@ -146,7 +146,7 @@ self.usernameOutlet.rx.text
         let loadingValue = Availability.pending(message: "Checking availability ...")
 
         // This will fire a server call to check if the username already exists.
-        // Its type is `Observable<ValidationResult>`
+        // Its type is `Observable<Availability>`
```

Comment:
The fix below is correct, but this one isn't - 
I believe its better to fix this to "Its type is `Observable<Bool>`" - because the `map` instantly askes `if available` at the entry point.

Also this example is a tad weird - because instead of `map` + `switchLatest` we usually use `flatMapLatest`... might need a tiny update

Reply:
Oh, right. Thanks.

I'll change this one to `Observable<Bool>`. But I think I'd better leave the `flatMapLatest` update for you.

Reply:
Yup, that's 💯% fine :) 
Thanks!